assets/section-main-product.css

```diff
@@ -38,25 +38,25 @@
 @media screen and (min-width: 990px) {
   .product--large:not(.product--no-media) .product__media-wrapper {
     max-width: 65%;
-    width: calc(65% - 1rem / 2);
+    width: calc(65% - var(--grid-desktop-horizontal-spacing) / 2);
   }
 
   .product--large:not(.product--no-media) .product__info-wrapper {
     padding-left: 4rem;
     max-width: 35%;
-    width: calc(35% - 1rem / 2);
+    width: calc(35% - var(--grid-desktop-horizontal-spacing) / 2);
   }
 
   .product--medium:not(.product--no-media) .product__media-wrapper,
   .product--small:not(.product--no-media) .product__info-wrapper {
     max-width: 55%;
-    width: calc(55% - 1rem / 2);
+    width: calc(55% - var(--grid-desktop-horizontal-spacing) / 2);
   }
 
   .product--medium:not(.product--no-media) .product__info-wrapper,
   .product--small:not(.product--no-media) .product__media-wrapper {
     max-width: 45%;
-    width: calc(45% - 1rem / 2);
+    width: calc(45% - var(--grid-desktop-horizontal-spacing) / 2);
```

Comment:
I was thinking more about this and wondering if we really want the grid padding on these 2 columns to be affected by the setting. It currently inherits the gaps from `.grid`, but maybe we should have a fixed spacing.

If we set the Horizontal Spacing to the maximum value (`40px`) and the Product Media size to `Large`, there'll be A LOT of whitespace between the two columns and little available space for product information:
![image](https://user-images.githubusercontent.com/37168033/149508359-f826e4f6-4c74-4312-86ea-698c3a7198ba.png)

I've noticed that other two-column layouts, such as `Image with Text` aren't affected by this setting, so it would also make sense to have this same behaviour on the product page.

Reply:
cc @melissaperreault. Thoughts on this? Should the gap between these two columns be controlled by the same `Layout > Horizontal spacing` setting?

Reply:
@LucasLacerdaUX  Do you suggest we reduce the value when the image is set to large?